### PR TITLE
Polishing repo

### PR DESCRIPTION
**Pull request by AlekseySh**

### CODE REVIEW

**Comment by AlekseySh** [comment]
Changelog:
-  removed unused gpus parameters from the configs

File: examples/cars/configs/val_cars.yaml [hunk 1]
```diff
@@ -29,9 +29,6 @@ metric_args:
 
 log_only_main_category: False
 
-gpus:
```
Comment (b0nce) [inline examples/cars/configs/val_cars.yaml:32]: Maybe we need defaults instead of `gpus`?

File: oml/utils/dataframe_format.py [hunk 2]
```diff
@@ -82,8 +82,10 @@ def check_retrieval_dataframe_format(
         )
 
         bboxes_df = df[~(df[X1_COLUMN].isna())]
-        assert all((bboxes_df[X1_COLUMN] < bboxes_df[X2_COLUMN]).to_list())
-        assert all((bboxes_df[Y1_COLUMN] < bboxes_df[Y2_COLUMN]).to_list())
+        mask_good_x1_x2 = bboxes_df[X1_COLUMN] < bboxes_df[X2_COLUMN]
+        mask_good_y1_y2 = bboxes_df[Y1_COLUMN] < bboxes_df[Y2_COLUMN]
+        assert all(mask_good_x1_x2.to_list()), (~mask_good_x1_x2).sum()
```
Comment (b0nce) [inline oml/utils/dataframe_format.py:87]: Assertation message?

Comment (AlekseySh) [reply oml/utils/dataframe_format.py:87]: added

**Review by DaloroAT — APPROVED** [review]

**Pull request merged**